README.md

```diff
@@ -9,4 +9,4 @@ To contribute, [pick an issue or create a new issue](https://github.com/kylebenn
 
 When your work is ready to be reviewed, open a Pull Request to merge your branch into dev and fill out the PR template.
 
-You will need to [install the statRdaysCFB package](https://github.com/kylebennison/statRdaysCFB) in order to access data and run most scripts. You will also need a CFBdata [API key](https://collegefootballdata.com/key).
\ No newline at end of file
+You will need to [install the statRdaysCFB package](https://github.com/kylebennison/statRdaysCFB) in order to access data and run most scripts. You will also need a free CFBdata [API key](https://collegefootballdata.com/key).
```

Comment:
this is what a comment would look like.

You can add comments in-line in code or generally with your review, and then you can approve, request changes, or just comment.